### PR TITLE
chore: offline package not upload aws s3

### PR DESCRIPTION
**Pull request by purelind**
The `download.pingcap.org` domestic and foreign DNS are currently pointing to the qiniu cloud, so in the future do not need to upload to aws s3 offline package.

### CODE REVIEW

File: jenkins/pipelines/cd/tiup/tiup-package-offline-mirror-dm.groovy [hunk 1]
```diff
@@ -0,0 +1,70 @@
+if(DEBUG_MODE == "true"){
```
Comment (wuhuizuo) [inline jenkins/pipelines/cd/tiup/tiup-package-offline-mirror-dm.groovy:1]: The file is about the the motivation？

Comment (lijie0123) [reply jenkins/pipelines/cd/tiup/tiup-package-offline-mirror-dm.groovy:1]: the file is added to CI from jenkins job script

**Review by lijie0123 — APPROVED** [review]

**Pull request merged**